schema@v1.1.1/dataset.json

```diff
@@ -11,7 +11,7 @@
   "properties": {
     "path": {
       "type": "string",
-      "format": "uri"
+      "format": "uri-reference"
```

Comment:
The commit says what you did, but does not explain the why? Why should it be `uri-reference` instead of `uri`?

Could you update the commit message with that info. Should help the Git archeologists, such as myself, that tend to look up the reason behind certain changes to understand the code base better.

Reply:
You should probably update the spec (`docs/ams-schema-spec.bs`) as well.

... and generate the HTML as well. The HTML will be auto published.

Reply:
I wanted to squash this commit into 'AB#5522 Add path property to dataset schema ' before rebasing.

Reply:
Thanks, I keep forgetting the spec. Will update them.